<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>941</width>
      <height>521</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../imagens/tela_adicionar.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_adicionar">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>450</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 0);
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>ADICIONAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_retornar">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>450</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 0);
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>RETORNAR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_cod_aero">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 0);
border-radius: 3px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_data_valid">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 0);
border-radius: 3px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_operador">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 0);
border-radius: 3px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_doc">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 0);
border-radius: 3px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_tipo">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>390</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 0);
border-radius: 3px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_fabricante">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>450</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 0);
border-radius: 3px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_modelo">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>160</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 0);
border-radius: 3px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_num_serie">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>220</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 0);
border-radius: 3px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_peso_max">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>280</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 0);
border-radius: 3px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_local">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>340</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 0);
border-radius: 3px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_ramo_ativ">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>400</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 0);
border-radius: 3px;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
